github-actions/get-variables.js

```diff
@@ -34,8 +34,8 @@ module.exports = (context) => {
     return {
         ZTOR_PR_CONTAINER_APP_NAME:  `${containerAppBaseName}-${versionIdentifier}`,
         VERSION_TAG: `${versionIdentifier}-${shortCommit}`,
-        GITHUB_ENVIRONMENT: getEnvironment(context),
-        GITHUB_SWARM_ENVIRONMENT: `swarm-${getEnvironment(context)}`,
+        ENVIRONMENT: getEnvironment(context),
+        GITHUB_ENVIRONMENT: `swarm-${getEnvironment(context)}`,
```

Comment:
it was a bit confusing the name changes, but it was replaced correctly. The names before were more clear.